config.toml

```diff
@@ -0,0 +1,18 @@
+baseURL = 'https://maplibre.org/'
+languageCode = 'en-us'
+title = 'MapLibre'
+
+
+
+[menu]
+  [[menu.main]]
+    identifier = "community"
+    name = "Community"
+    url = "https://maplibre.org/community"
+    weight = 40
+
+
+[params]
+  key = "R8AisQ0L6EL1wW8JVYQ"
+  description = "Tesla's Awesome Hugo Site"
+  keywords = ""
```

Comment:
those red marks, bleh - most editors have a setting to add a newline at the end of a file (and remove mulitple empty newlines at the end as well)

Reply:
I enabled the VSCode option for the project. I also added pretter which will format the whole project.